src/app/_core/services/weather.service.ts

```diff
@@ -96,7 +96,7 @@ export class WeatherService {
       })
   }
 
-  /* HTTP CALLS */
+  /***** Angular HttpClientModule for API calls *****/
```

Comment:
- Not relevant comment
- Wrong syntax. Must be like this (depending of your params):
![image](https://user-images.githubusercontent.com/37407695/178761453-aa46a3d4-8c7c-4afe-920d-645f6199d246.png)